skills/csharp/experimental/newsskill/Responses/Main/MainStringsTexts.lg

```diff
@@ -35,4 +35,7 @@
 - What country or region are you in?
 
 # MarketRetryPrompt.Text(Data)
-- Couldn't find that country or region. What country or region do you want to search in?
\ No newline at end of file
+- Couldn't find that country or region. What country or region do you want to search in?
+
+# FeatureNotAvailable.Text(Data)
```

Comment:
Translate for other languages? Or maybe we can do this later.

Reply:
good catch. fixed.